src/modules/uORB/Subscription.cpp

```diff
@@ -64,12 +64,6 @@ bool Subscription::subscribe()
 			if (node != nullptr) {
 				_node = node;
 				_node->add_internal_subscriber();
-
-				const unsigned curr_gen = _node->published_message_count();
```

Comment:
These codes can be deleted, _last_generation is 0 by default and there is no problem, uORB::DeviceNode will automatically set it to a valid value.